File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/get-game-result/get-game-result-response-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';
import type { SuitType } from '../../enums/suit-type-enum';

export interface GetGameResultResponseDto extends BaseDto {
    /** Место, которое заняла лошадь в игре */
    Place: number;
    /** Масть */
    BetSuit: SuitType;
    /** Код игры */
    GameId: string;
    /** Код пользователя */
    UserId: string;
    /** Полное имя пользователя */
    FullName: string;
}

File: HorseRacing.Contracts/Models/Game/Responses/GetGameResult/GetGameResultResponseDto.cs
using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Domain.GameAggregate.Enums;

namespace HorseRacing.Contracts.Models.Game.Responses.GetGameResult
{
    public class GetGameResultResponseDto : BaseDto
    {
        /// <summary>
        /// Место, которое заняла лошадь в игре
        /// </summary>
        public int Place { get; set; }
        /// <summary>
        /// Масть
        /// </summary>
        public SuitType BetSuit { get; set; }
        /// <summary>
        /// Код игры
        /// </summary>
        public Guid GameId { get; set; }
        /// <summary>
        /// Код пользователя
        /// </summary>
        public Guid UserId { get; set; }
        /// <summary>
        /// Полное имя пользователя
        /// </summary>
        public string FullName { get; set; } = string.Empty;
    }
}
